openpoke/decidim
# Sticky header hide and show on scroll and visible login access

#### :tophat: What? Why?
Sticky header hide and show on scroll and visible login access

#### :pushpin: Related Issues
- Related to #12868, #12871
- Fixes #?

#### Testing
You can visit with a mobile browser.
- The sticky header hides and displays if the user scrolls down and then scrolls up. 
- See the header, which has visible login access.
- Click the menu.
- The menu has a search and a language chooser.

### :camera: Screenshots
![image](https://github.com/openpoke/decidim/assets/116598037/a357e478-0fc1-4e34-a1c8-d929f0b498b7)
header default
![image](https://github.com/openpoke/decidim/assets/116598037/f87d4b65-76ff-4468-8cc1-c20dfcf7be9c)
header login photo
![image](https://github.com/openpoke/decidim/assets/116598037/b04f5ee3-e56d-44dc-ae14-feca8f3d9346)
add search added to the menu and language chooser to dropdown menu

:hearts: Thank you!
